src/transformers/models/speech_to_text_2/configuration_speech_to_text_2.py

```diff
@@ -68,8 +68,6 @@ class Speech2Text2Config(PretrainedConfig):
             for more details.
         use_cache (`bool`, *optional*, defaults to `True`):
             Whether or not the model should return the last key/values attentions (not used by all models).
-        max_source_positions (`int`, *optional*, defaults to 6000):
```

Comment:
`Speech2Text2` only implements decoder models (`Speech2Text2Decoder`, `Speech2Text2DecoderWrapper` and  `Speech2Text2ForCausalLM`), and `max_source_positions` is for `Encoder`, like `Speech2TextEncoder`.